docs/command/mongocli-atlas-privateEndpoints-gcp-create.txt

```diff
@@ -0,0 +1,74 @@
+.. _mongocli-atlas-privateEndpoints-gcp-create:
+
+==========================================
+mongocli atlas privateEndpoints gcp create
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create a new GCP private endpoint for your project.
+
+Syntax
+------
+
+.. code-block::
+
+   mongocli atlas privateEndpoints gcp create [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for create
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file.
+   * - --projectId
+     - string
+     - false
+     - Project ID to use. Overrides the settings in the configuration file or environment variable.
+   * - --region
+     - string
+     - true
+     - Physical location of your MongoDB cluster. For a complete list of supported AWS regions, see: https://docs.atlas.mongodb.com/reference/amazon-aws/#amazon-aws. For a complete list of supported Azure regions, see: https://docs.atlas.mongodb.com/reference/microsoft-azure/#microsoft-azure. For a complete list of supported GCP regions, see: https://docs.atlas.mongodb.com/reference/google-gcp/#google-gcp.
```

Comment:
[q] is this description correct? this is the description for cluster regions, just making sure you can use the same

Reply:
yes, you are right. My fault here as Bianca added correctly `PrivateEndpointRegion` before my review. Not seeing the flag matching the usage tricks me

Reply:
thanks! I'll move back to PrivateEndpointRegion